lntest/itest/lnd_multi-hop_test.go

```diff
@@ -255,6 +261,15 @@ func createThreeHopNetwork(t *harnessTest, net *lntest.NetworkHarness,
 		t.Fatalf("unable to connect bob to carol: %v", err)
 	}
 
+	// Make sure Carol has enough utxos for anchoring.
+	for i := 0; i < 2; i++ {
```

Comment:
maybe add a small explanation why we need to have separate UTXOs for this to work

Reply:
added